posthog/models/person/util.py

```diff
@@ -162,6 +162,26 @@ def get_persons_by_uuids(team: Team, uuids: List[str]) -> QuerySet:
 
 
 def delete_person(person: Person) -> None:
+    def _get_distinct_ids_with_version() -> Dict[str, int]:
```

Comment:
It's uncommon to create nested functions like this in python. Could we move this to the top level?